lib/digsync_web/live/groups_live.html.heex

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
we dont want these doctype html stuff in every liveview, you can remove. we are probably gonna pull it into a layout at some point